### PR TITLE
to flyteorg

### PR DESCRIPTION
**Pull request by brucearctor**
# TL;DR
Renaming/moving refs to flyteorg from lyft

## Type
 - [x] Bug Fix/Tech-Debt

## Are all requirements met?

 - [x] Code completed ??

### CODE REVIEW

**Comment by brucearctor** [comment]
NOT OK TO MERGE.

Tests passed because I removed the 'goimports' from the linter.

https://github.com/flyteorg/flytectl/pull/43/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9

I'm not familiar enough (yet) to understand what specifically from that is causing the failure :-(  ... 

Also, is unclear ( I need to do more HW to dig into) whether/how can pull up logs from prior build runs for understanding the error messages.  Also, TODO for me is setting up so I can run the builds on my machine (can I?) before pushing?

File: go.mod [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.7.0
 	github.com/yalp/jsonpath v0.0.0-20180802001716-5cc68e5049a0
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975 // indirect
-	google.golang.org/grpc v1.26.0
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/apimachinery v0.18.3 // indirect
+	google.golang.org/grpc v1.35.0
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/client-go v11.0.0+incompatible // indirect
```
Comment (kumare3) [inline go.mod:22]: Why do we need k8s client?
And if so, can we update it to - https://github.com/flyteorg/flytepropeller/blob/master/go.mod#L30

I feel we do not need it - lets delete it?

**Comment by kumare3** [comment]
Couple comments, then we should be good to go
you can also run
`go mod tidy`

File: go.mod (hunk 1 above)
Comment (brucearctor) [reply go.mod]: Doesn't look like was needed.  Is Deleted.

**Comment by brucearctor** [comment]
Not just `go mod tidy` ...  Also required `go fmt go fmt github.com/flyteorg/flytectl/...` to fmt all files, specifically wasn't the formatting but ordering of imports was breaking the linter.  🤞 that the checks pass right now

**Comment by brucearctor** [comment]
@kumare3 -- I think good to go

**Comment by brucearctor** [comment]
See: https://github.com/flyteorg/flyte/issues/729

File: go.sum [hunk 2]
```diff
@@ -5,59 +5,109 @@ cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6A
 cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
 cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
 cloud.google.com/go v0.46.3/go.mod h1:a6bKKbmY7er1mI7TEI4lsAkts/mkhTSZK8w33B4RAg0=
-cloud.google.com/go v0.52.0 h1:GGslhk/BU052LPlnI1vpp3fcbUs+hQ3E+Doti/3/vF8=
+cloud.google.com/go v0.50.0/go.mod h1:r9sluTvynVuxRIOHXQEHMFffphuXHOMZMycpNR5e6To=
```
Comment (kumare3) [inline go.sum:8]: we need to run go mod tidy, once, but that is fine

**Review by kumare3 — APPROVED** [review]

**Comment by kumare3** [comment]
I have no idea, why this commit is blocked, I am merging it as it is blocking some other folks

**Pull request merged**